app/models/sbom-vulnerability-audit.ts

```diff
@@ -12,6 +20,81 @@ export default class SbomVulnerabilityAuditModel extends Model {
 
   @attr()
   declare versions: VulnerabilityAuditVersion[];
+
+  get parsedVersions() {
+    type FinalReturnType = {
```

Comment:
Not a blocker but I think this type declaration would do well at the top of this file.

Reply:
done